ci/ray_ci/test_tester.py

```diff
@@ -35,7 +35,7 @@ def test_get_container() -> None:
         "ci.ray_ci.tester_container.TesterContainer.install_ray",
         return_value=None,
     ):
-        container = _get_container("core", 3, 1, 2, 0)
+        container = _get_container("core", 3, 1, 2, "linux", 0)
```

Comment:
this has so many args now, could you add arg keys here to make is more readable?